models/event-videos-post.json

```diff
@@ -11,6 +11,16 @@
       "description": "The unique url of the video. Currently only Youtube and Vimeo sources are allowed.",
       "pattern": "^http(s?):\\/\\/(www\\.)?((youtube\\.com\\/watch\\?v=([^\\/#&?]*))|(vimeo\\.com\\/([^\\/#&?]*)))"
     },
+    "language": {
+      "type": "string",
+      "example": "nl",
+      "enum": [
+        "nl",
+        "fr",
+        "de",
+        "en"
+      ]
+    },
```

Comment:
Nit: A `description` could help to give some more context about what the purpose of this property is. (Like the other properties also have)